canonical-web-and-design/jaas-dashboard
# Restyle status strip

## Done

- Restyle status strip by making one component to use across models and model details pages
- Driveby: fix leaky empty relations error message

## QA

- Pull code
- Run `./run clean && ./run serve`
- Open http://0.0.0.0:8036/models
- Verify counter displays as expected
- Click through to a model details
- Verify counter displays as expected

@long-chung Please play with screen resolutions, there is currently some wrapping on models with lots of different statuses

## Details

Fixes https://github.com/canonical-web-and-design/juju-squad/issues/1507
